dcrtimed/backend/filesystem/filesystem.go

```diff
@@ -198,7 +198,6 @@ func (fs *FileSystem) flush(ts int64) error {
 	}
 
 	hashes := make([]*[sha256.Size]byte, 0, 4096)
-
```

Comment:
There are a few random empty lines that are deleted in the PR.  Please put them back in.